compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeBridge.desktop.kt

```diff
@@ -86,16 +94,30 @@ internal abstract class ComposeBridge(
 
     private var window: Window? = null
 
+    private fun refocus() {
+        if (component.isFocusOwner) {
+            _invisibleComponent.requestFocusTemporary()
+            component.requestFocus()
+        }
+    }
+
     private val platformComponent: PlatformComponent = object : PlatformComponent {
         override fun enableInput(inputMethodRequests: InputMethodRequests) {
             currentInputMethodRequests = inputMethodRequests
             component.enableInputMethods(true)
-            val focusGainedEvent = FocusEvent(focusComponentDelegate, FocusEvent.FOCUS_GAINED)
-            component.inputContext.dispatchEvent(focusGainedEvent)
+            // Without resetting the focus, Swing won't update the status (doesn't show/hide popup)
```

Comment:
Is it possible to cover such a case with a test?

Reply:
There 2 useful tests that could be written:

1. Checking that some Swing component enables input methods when we focus on a textfield. Unfortunately, Swing doesn't provide any API for that.

2. Checking Chinese input in integration with Chinise input enabled in the system. But for that we need a Docker container, a task on CI, and a special test annotation @EnvironmentTest.

Other kind of tests aren't useful:
1. Checking that TextField enables Input methods on ComposeScene level. It isn't very needed right now, as a similar test already exists in Jetpack Compose (that checks that we call appropriate `PlatformTextInputService` methods). 
2. Injecting fake JComponent, on which we check that we call `enableInputMethods`. But this bounds the test to the implementation details.